src/sql/workbench/contrib/views/browser/treeView.ts

```diff
@@ -670,6 +670,7 @@ class TreeDataSource implements IAsyncDataSource<ITreeItem, ITreeItem> {
 		private treeView: ITreeView,
 		private id: string,
 		private withProgress: <T>(task: Promise<T>) => Promise<T>,
+		@IConfigurationService private readonly configurationService: IConfigurationService,
```

Comment:
The configurationService is what holds our user configuration. We'll be adding it some classes, and as parameters to some methods.